Docs for retriggering through issue comment (#567)
packit/packit.dev: content/docs/guide.md

Take a look at the tables [here](/docs/actions/) to know what actions are available for each job.

### Users

> Packit can check for the user's rights to complete a job (this depends on the job type).

Reviewer: ```suggestion
Packit checks if the user has sufficient permissions to complete a job (this depends on the job type).
```

For this reason the profile of a Packit user has to be accessible.

> In Github, as an example, this means that the membership of a user to its team has to be public (https://github.com/orgs/[org_name]/people/[user_name]).

Reviewer: ```suggestion
For example, on Github this means that the membership of a user to their team has to be public (see https://github.com/orgs/[org_name]/people/[user_name]).
```

> For example, on Github this means that the membership of a user to their team has to be public (see https://github.com/orgs/[org_name]/people/[user_name]).

Reviewer: ```suggestion
For example, on Github this means that the membership of a user to their team has to be public (see https://github.com/orgs/[org_name]/people/[user_name] ).
```
One last suggestion. Without the space, hugo renders the `)` as a part of the link.
